lib/kafo/kafo_configure.rb

```diff
@@ -291,7 +291,7 @@ def set_parameters
 
     def set_app_options
       self.class.app_option ['--[no-]colors'], :flag, 'Use color output on STDOUT',
-                            :default => !!config.app[:colors]
+                            :default => Kafo::ColorScheme.colors_possible?
```

Comment:
@ekohl We can just set the default here passed on the computed value with no need to persist this between runs, that way a user can specify this at runtime if they desire to override or let the app decide if it's possible.

This worked for me with `--no-colors` being passed.

Reply:
This removes the option for a scenario to always disable colors, which was previously possible.

Reply:
I put it back the way it has been. Let's just leave colors as an option alone the way it worked before.